src/Symfony/Bundle/FrameworkBundle/Translation/PhpExtractor.php

```diff
@@ -37,7 +39,29 @@ class PhpExtractor extends AbstractFileExtractor implements ExtractorInterface
      *
      * @var array
      */
-    protected $sequences = array(
+    protected static $sequences = array(
+        array(
+            '->',
+            'trans',
+            '(',
+            self::MESSAGE_TOKEN,
+            ',',
+            self::MESSAGE_PARAMS_TOKEN,
+            ',',
+            self::DOMAIN_TOKEN,
+        ),
+        array(
+            '->',
+            'transChoice',
+            '(',
+            self::MESSAGE_TOKEN,
+            ',',
+            T_LNUMBER,
```

Comment:
Okay, I added this lines and the Tests don't pass. Thank you!
I will fix it asap.